mysql-test/suite/innodb/r/percona_log_encrypt_change_mk.result

```diff
@@ -0,0 +1,17 @@
+call mtr.add_suppression("Redo log encryption mode can't be switched without stopping the server and recreating the redo logs");
+SELECT @@innodb_redo_log_encrypt;
+@@innodb_redo_log_encrypt
+master_key
+SET GLOBAL innodb_redo_log_encrypt=KEYRING_KEY;
+Warnings:
+Warning	1210	 Redo log encryption mode can't be switched without stopping the server and recreating the redo logs.
+SELECT @@innodb_redo_log_encrypt;
+@@innodb_redo_log_encrypt
+master_key
+SET GLOBAL innodb_redo_log_encrypt=OFF;
+SET GLOBAL innodb_redo_log_encrypt=KEYRING_KEY;
```

Comment:
THis is supposed to throw warning right? Current is MK. MK->Keying warning is there
MK to OFF (no warning) and after OFF, to KEYRING_KEY (with MK existing) should throw warning right?

Reply:
Ops, and it does - but it passes the first check in ha_innodb, and the second check in srv0srv places the warning in the error log. I'll fix this